### PR TITLE
Parallelizes auctions network calls.

### PR DESCRIPTION
**Pull request by ashfurrow**
This is a replacement PR for #2276, due to some problems I was having with Metaphysics I figured in the mean time we can just run these network requests in parallel to get a significant speed boost.

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">There were <a href="https://eigen-ci.s3.amazonaws.com/4037/index.html">snapshot errors</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_circle">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by orta** [comment]
Ah yeah, this is nice! 👍 

I feel like I saw this error before>? Did that PR get merged?

**Comment by ashfurrow** [comment]
Yeah, it's intermittent since the recent onboarding changes, going to take a look monday.

**Comment by ashfurrow** [comment]
Right now, master is failing with the same error. I'm in favour of merging this as-is, and I'll investigate master separately.

**Comment by ashfurrow** [comment]
Okay, this incidentally "fixes the test" and should be 🍏

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -597,7 +597,6 @@
 		60CEA77E19B61F8000CC3A91 /* ARArtistNetworkModel.m in Sources */ = {isa = PBXBuildFile; fileRef = 60CEA77D19B61F8000CC3A91 /* ARArtistNetworkModel.m */; };
 		60CF980817BF79CA005ED59B /* ARArtistBiographyViewController.m in Sources */ = {isa = PBXBuildFile; fileRef = 60CF980717BF79CA005ED59B /* ARArtistBiographyViewController.m */; };
 		60D70DE41D01F420008888D1 /* LiveAuctionLotViewModelSpecs.swift in Sources */ = {isa = PBXBuildFile; fileRef = 60D70DE31D01F420008888D1 /* LiveAuctionLotViewModelSpecs.swift */; };
-		60D7FB731AC35F5600F5043A /* ARPersonalizeWebViewControllerTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 60D7FB721AC35F5600F5043A /* ARPersonalizeWebViewControllerTests.m */; };
```
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: `ARPersonalizeWebViewController` is not used in the app 👍

**Pull request merged**